doc/examples/filters/plot_homomorphic.py

```diff
@@ -0,0 +1,105 @@
+"""
+=====================
+Homomorphic Filtering
+=====================
+
+Homomorphic filtering can be used to reduce variations in illumination from an
+image. Specifically, consider an image model where the observed `image` is the
+product of a desired image, `im_true` and some spatially varying
```

Comment:
```suggestion 
product of a desired image, `im_true`, and some spatially varying
```

Reply:
+1
When resolving conflicts in `skimage/filters/_fft_based.py`, I fixed similar commas, which should either come in pairs or be disposed of... I went for the latter because I felt the text was meant to be short and concise (docstring vs. tutorial).